docs/licensing_faq.md

```diff
@@ -1,35 +1,41 @@
-VCA Licensing FAQ
-==================
```

Comment:
@vigneshvijay94 where did you get this text from? Did a lawyer write or check this? Because that stuff is complicated.